packages/vue-ds/src/components/Pagination/Pagination.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <section v-if="!isInvalid" :class="PaginationClass">
+    <div :class="PaginationContainerClass">
+      <button
+        v-if="hasPreviousCondition"
```

Comment:
man, `v-if` is a really nice directive, but it remove and add markup at DOM, maybe in this case you can use v-show instead. `v-show` works like `display: none` 😬😬🚀🚀

Reply:
wow it's a really good approach! I'll change it 😄